docs/UserGuide.md

```diff
@@ -124,7 +124,7 @@ Beyond individual products, you can also:
   e.g. in `addClient n/NAME ...`, `NAME` is a parameter which can be used as `addClient n/John Tan ...`.
 
 * Items in square brackets are optional, while those not in square brackets are compulsory.<br>
-  e.g `addClient n/NAME p/PHONE_NUMBER [e/EMAIL]` can be used as `addClient i/1 p/12345678 e/John@abc.com a/123 Avenue b/12122000` or as `addClient n/John Tan p/12345678`.
+  e.g `addClient n/NAME p/PHONE_NUMBER [e/EMAIL]` can be used as `addClient n/John Tan p/12345678 e/johntan@insurec.com` or as `addClient n/John Tan p/12345678`.
 
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
```

Comment:
I believe this one i done it already, hence there maybe conflicts that you need to resolve. My pr has already been merged. So you can use it to resolve the conflict.

Reply:
There is an error here, addClient doesn't take prefix `i/`, so going to change that part.